R/features.R

```diff
@@ -414,6 +414,337 @@ setMethod("calculatePeakQualities", "features", function(obj, weights, flatnessF
 })
 
 
+#' @describeIn features Obtain the total ion chromatogram/s (TICs) of the analyses.
+#' @param msLevels Integer vector with the ms levels (i.e., 1 for MS1 and 2 for MS2) 
+#' to obtain TIC traces.
+#' @aliases getTICs
+#' @export
+setMethod("getTICs", "features", function(obj, i = NULL, retentionRange = NULL, msLevels = 1)
```

Comment:
I'm not sure if the `i` argument is useful here (you can just subset the `features` object when calling this function. The `plotChroms()` method does have this option though, mainly for speed. If you want to keep it then maybe rename `i` to `analysis` for consistency.

Reply:
Removed, not needed indeed.